qdrant/deployment/src/test/java/io/quarkiverse/langchain4j/qdrant/QdrantEmbeddingStoreTest.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkiverse.langchain4j.qdrant;
+
+import jakarta.inject.Inject;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.Asset;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.AllMiniLmL6V2QuantizedEmbeddingModel;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreWithoutMetadataIT;
+import dev.langchain4j.store.embedding.qdrant.QdrantEmbeddingStore;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class QdrantEmbeddingStoreTest extends EmbeddingStoreWithoutMetadataIT {
+
+    public static final String COLLECTION_NAME = "qdrant_test_embeddings";
+
+    @RegisterExtension
+    static final QuarkusUnitTest unitTest = new QuarkusUnitTest()
+            .setArchiveProducer(QdrantEmbeddingStoreTest::archive);
+
+    @Inject
+    QdrantEmbeddingStore embeddingStore;
+
+    private final EmbeddingModel embeddingModel = new AllMiniLmL6V2QuantizedEmbeddingModel();
+
+    @Override
+    protected void clearStore() {
+        embeddingStore.clearStore();
+    }
+
+    @Override
+    protected EmbeddingStore<TextSegment> embeddingStore() {
+        return embeddingStore;
+    }
+
+    @Override
+    protected EmbeddingModel embeddingModel() {
+        return embeddingModel;
+    }
+
+    private static JavaArchive archive() {
+        Asset properties = new StringAsset(String.join("\n",
+                "quarkus.langchain4j.qdrant.devservices.collection.name=" + COLLECTION_NAME,
+                // TODO: as today the dev service leverages the service name as a name of the collection, find a better way
+                "quarkus.langchain4j.qdrant.devservices.service-name=" + COLLECTION_NAME,
```

Comment:
The `QdrantEmbeddingStore` does not auto create the collection hence at this stage I set it to be the same as the service name. Ideally it should use the same value as `quarkus.langchain4j.qdrant.devservices.collection.name` but I haven't found a way to get such value at build time (long time since I did a quarkus extension)

Reply:
You can't :)

Reply:
ok :) so you are ok to use the service name ? or the creation of the collection should be moved to runtime ?

Reply:
> ok :) so you are ok to use the service name

Sure, we do can do that.

Reply:
ok, so apart from the client discussion, this is likely ready for review